back/engines/commercial/project_folders/spec/policies/project_policy_spec.rb

```diff
@@ -5,19 +5,21 @@
 describe ProjectPolicy do
   subject { described_class.new(user, project) }
 
-  context 'for a project contained within a folder the user moderates' do
-    let!(:project) { create(:project, admin_publication_attributes: { parent_id: project_folder.admin_publication.id }) }
+  context 'for a continuous project contained within a folder the user moderates' do
```

Comment:
Shouldn't we add tests for timeline projects? Deleting inputs is not permitted, but that behaviour is not described here.